lib/rubocop/cop/style/operator_method_call.rb

```diff
@@ -38,10 +38,20 @@ def on_send(node)
 
         private
 
-        def parenthesized_argument?(argument)
+        # Checks for an acceptable case of `foo.+(bar).baz`.
+        def method_call_to_parenthesized_arg?(argument)
```

Comment:
I think it's better to use `with` instead of `to` here, as you don't really call "to" arguments, but rather "with" them.